### PR TITLE
Add DocuSign Monitor connector

### PR DESCRIPTION
**Pull request by sachinira**
# Description
Generate the DocuSign Monitor connector using the official OpenAPI specification. 
https://developers.docusign.com/docs/monitor-api/

Fixes: https://github.com/ballerina-platform/ballerina-extended-library/issues/248

One line release note: 
- Generate DocuSign Monitor connector using Open API specification.

## Type of change

Please delete options that are not relevant.
- [x] New feature (non-breaking change which adds functionality)

# How Has This Been Tested?
Not tested yet

**Test Configuration**:
* Ballerina Version: SL Beta 6
* Operating System: Ubuntu 20.04
* Java SDK: 11

# Checklist:

### Changes to OpenAPI definition

- [x] Modified `info` section of the OpenAPI definition file - [Example](https://github.com/ballerina-platform/ballerina-extended-library/discussions/74)

    - `description` - what the connector is about 
    - `x-ballerina-init-description` OpenAPI extension - connector initialization details 
    - `x-ballerina-display` OpenAPI extension - connector name and path to the connector icon 

### Changes to connector module

- [x] Added license header at the top of each .bal file. 
- [x] Added Package.md as per the guide [here](https://github.com/ballerina-platform/ballerina-extended-library/discussions/77)
- [x] Added Module.md for the module of the  connector as per the guide [here](https://github.com/ballerina-platform/ballerina-extended-library/discussions/78)
- [x] Added connector icon to the root of the connector. Icon needs to be a `png` of 200x200 px size with name `icon.png`
- [x] Added Ballerina.toml file with following information. For keywords refer to [guide](https://github.com/ballerina-platform/ballerina-extended-library/discussions/72)

### Verifying connector module

- [x] Compiled the connector successfully with the targeted ballerina version. 
- [ ] Conducted smoke tests on the connector (Recommended when OpenAPI definition is obtain from an unofficial source)

### Security checks
 - [x] Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? 
 - [x] Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets?

### CODE REVIEW

**Review by aneeshafedo — COMMENTED** [review]
Missing `Dependencies.toml`

File: openapi/docusign.monitor/Ballerina.toml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+[package]
+license = ["Apache-2.0"]
+keywords = ["Content & Files/Documents", "Cost/Freemium"]
+org = "ballerinax"
+name = "docusign.monitor"
+distribution = "slbeta6"
+repository = "https://github.com/ballerina-platform/ballerinax-openapi-connectors"
+version = "0.1.0"
```
Comment (RolandHewage) [inline openapi/docusign.monitor/Ballerina.toml:8]: Shall we update the version to 1.0.0

File: openapi/docusign.monitor/openapi.yaml [hunk 2]
```diff
@@ -0,0 +1,121 @@
+openapi: 3.0.1
+info:
+  title: Monitor API
+  description: DocuSign Monitor helps organizations protect their agreements with round-the-clock activity tracking.
```
Comment (RolandHewage) [inline openapi/docusign.monitor/openapi.yaml:4]: Shall we follow the standard used in other connectors in the description as follows
https://github.com/ballerina-platform/openapi-connectors/blob/49164bb7baae2ddcd19037c109548143c0fe32d7/openapi/ably/openapi.yaml#L14
https://github.com/ballerina-platform/openapi-connectors/blob/49164bb7baae2ddcd19037c109548143c0fe32d7/openapi/spotify/openapi.yaml#L7

**Review by RolandHewage — COMMENTED** [review]
Requested few changes

**Review by RolandHewage — APPROVED** [review]

**Review by aneeshafedo — APPROVED** [review]

**Review by abeykoon — APPROVED** [review]

**Pull request merged**